mlir/lib/Dialect/SPIRV/IR/AtomicOps.cpp

```diff
@@ -42,8 +42,9 @@ static LogicalResult verifyAtomicUpdateOp(Operation *op) {
                              << stringifyTypeName<ExpectedElementType>()
                              << " value, found " << elementType;
 
+  StringRef semanticsAttrName = spirv::AtomicAndOp::getSemanticsAttrName(op->getName()).strref();
```

Comment:
I don't quite get why you use StringRef here?
Can we stick to StringAttr instead? This will make it use pointer comparison instead of string comparisons.

Reply:
Ok, will do

Reply:
fixed